src/quo2/components/buttons/button.cljs

```diff
@@ -138,7 +138,8 @@
 
 (defn shape-style-container
   [type icon size]
-  {:border-radius (if (and icon (#{:primary :secondary :danger} type))
+  {:height        size
```

Comment:
can you see other components that are using this prop and just do a quick check that's it not having any unexpected consequences.

Reply:
I did, It shouldn't, in fact it should fix them

Reply:
> I did, It shouldn't, in fact it should fix them

Please can you provide an example, perhaps a screenshot.